site/gdocs/OwidGdoc.tsx

```diff
@@ -94,6 +94,7 @@ export function OwidGdoc({
                 homepageMetadata: get(props, "homepageMetadata", {}),
                 latestWorkLinks: get(props, "latestWorkLinks", []),
                 linkedChartViews: get(props, "linkedChartViews", {}),
+                tags: get(props, "tags", []) ?? [],
```

Comment:
We already pass the default value (`[]`) to `get`, so we don't need the `?? []`.